include/keystone-client.h

```diff
@@ -0,0 +1,200 @@
+#ifndef KEYSTONE_CLIENT_H_
```

Comment:
Missing licence on imported files

Reply:
Right!

Besides: Do you think that it is alright to include a source file like that? The repo says it is GPL v3.